### PR TITLE
pylightning: update getroute() parameters

### PR DESCRIPTION
**Pull request by billygarrison**
Removes `seed` and adds `maxhops`

### CODE REVIEW

File: contrib/pylightning/lightning/lightning.py [hunk 1]
```diff
@@ -302,13 +302,15 @@ def listnodes(self, node_id=None):
...
         """
         Show route to {id} for {msatoshi}, using {riskfactor} and optional
         {cltv} (default 9). If specified search from {fromid} otherwise use
         this node as source. Randomize the route with up to {fuzzpercent}
-        (0.0 -> 100.0, default 5.0) using {seed} as an arbitrary-size string
-        seed. {exclude} is an optional array of scid/direction to exclude.
+        (0.0 -> 100.0, default 5.0). {exclude} is an optional array of 
+        scid/direction to exclude. Limit the number of hops in the route to
+        {maxhops}.
+
```
Comment (rustyrussell) [inline contrib/pylightning/lightning/lightning.py]: flake8 is upset with the spaces on this line.  Maybe delete altogether?

File: contrib/pylightning/lightning/lightning.py [hunk 2]
```diff
@@ -302,13 +302,15 @@ def listnodes(self, node_id=None):
...
 
-    def getroute(self, node_id, msatoshi, riskfactor, cltv=9, fromid=None, fuzzpercent=None, seed=None, exclude=[]):
+    def getroute(self, node_id, msatoshi, riskfactor, cltv=9, fromid=None, fuzzpercent=None, exclude=[], maxhops=20):
         """
         Show route to {id} for {msatoshi}, using {riskfactor} and optional
         {cltv} (default 9). If specified search from {fromid} otherwise use
         this node as source. Randomize the route with up to {fuzzpercent}
-        (0.0 -> 100.0, default 5.0) using {seed} as an arbitrary-size string
-        seed. {exclude} is an optional array of scid/direction to exclude.
+        (0.0 -> 100.0, default 5.0). {exclude} is an optional array of
```
Comment (rustyrussell) [inline contrib/pylightning/lightning/lightning.py]: flake8 doesn't like ' ' at the end of this line, unfortunately.

**Comment by billygarrison** [comment]
Thanks @rustyrussell, I removed the extra whitespace from the docstring.

**Comment by cdecker** [comment]
ACK https://github.com/ElementsProject/lightning/pull/2478/commits/79d27703207ec1f460f356fc8d1762643afaf73e

**Pull request merged**